internal/driver/commands.go

```diff
@@ -128,7 +128,7 @@ var pprofCommands = commands{
 // reported generated by pprof.
 var pprofVariables = variables{
```

Comment:
Optional:
Consider modifying interactive help output to produce something like:

```
(pprof) help nodefraction
nodefraction=<f>
Hide nodes below <f>*total
```

instead of

```
(pprof) help nodefraction
Hide nodes below <f>*total
```

Reply:
Done.